tests/conftest.py

```diff
@@ -187,3 +188,31 @@ def pytest_collection_modifyitems(items):
             # has `@RunIf(slow=True)`
             if marker.name == "skipif" and marker.kwargs.get("slow")
         ]
+
+
+def pytest_addoption(parser):
+    parser.addoption("--hpus", action="store", type=int, default=1, help="Number of hpus 1-8")
+    parser.addoption(
+        "--hmp-bf16", action="store", type=str, default="./ops_bf16_mnist.txt", help="bf16 ops list file in hmp O1 mode"
+    )
+    parser.addoption(
+        "--hmp-fp32", action="store", type=str, default="./ops_fp32_mnist.txt", help="fp32 ops list file in hmp O1 mode"
+    )
+
+
+@pytest.fixture
+def hpus(request):
+    hpus = request.config.getoption("--hpus")
```

Comment:
Do you expect to run CI with different HPUs values? Or is this for debugging?

Reply:
We would like to keep this configurable for subsequent PR's so that some amount CI task parallelization can be done